### PR TITLE
Upgrade PA packages for OS3.0 compatibility

### PR DESCRIPTION
**Pull request by dzane17**
**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**

**Describe the solution you are proposing**

**Describe alternatives you've considered**

**Additional context**

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dzane17** [comment]
Fixes https://github.com/opensearch-project/performance-analyzer/issues/307

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -13,7 +13,6 @@ jobs:
     strategy:
       matrix:
         java:
-          - 11
```
Comment (kaushalmahi12) [inline .github/workflows/gradle.yml:16]: Is java 11 causing the issue here ?

Comment (kkhatua) [reply .github/workflows/gradle.yml:16]: Not sure, but unlikely. I see some of the build tasks by @kiranprakash154  having succeeded:
https://github.com/opensearch-project/performance-analyzer/actions/workflows/gradle.yml
The gradle tasks weren't skipped, so they do appear to have run normally.

Comment (dzane17) [reply .github/workflows/gradle.yml:16]: I don't think so. Bwc tests run with jdk11 and jdk17 every time and both fail. All the successful runs I saw in that list are on 2.x or 1.x branches which use old bwcVersion = 1.1 (changed to 2.1 now)

**Comment by dzane17** [comment]
Fixed as part of https://github.com/opensearch-project/performance-analyzer/pull/313

**Pull request closed without merging**